Traceback (most recent call last):
  File "/shared/home/asandri/MAHABIO_analysis/scripts/MAG_mapping/2-1-MAG_size_stat.py", line 24, in <module>
    stats_df_english = stats_df.rename(columns={
NameError: name 'stats_df' is not defined
